openstack/nova-hypervisor-agents/templates/hypervisors-kvm-daemonset.yaml

```diff
@@ -0,0 +1,296 @@
+            name: libvirt
+          - mountPath: /var/run
+            name: run
+          - mountPath: /etc/libvirt
+            name: etclibvirt
+          - mountPath: /etc/libvirt/libvirtd.conf
+            name: hypervisor-config
+            subPath: libvirtd.conf
+            readOnly: true
+          - mountPath: /container.init
+            name: bin
+      {{- end }}
+      dnsConfig:
+        searches:
+          - {{ .Values.global.region }}.{{ .Values.global.tld }}
+      volumes:
+{{- if not .Values.hypervisor_on_host }}
+      - name: instances
+        persistentVolumeClaim:
+          claimName: nova-kvm-shared
+      - name: libvirt
+        emptyDir:
+          medium: Memory
+      - name: run
+        emptyDir:
+          medium: Memory
+      - name: modules
+        hostPath:
+          path: /lib/modules
+      - name: etclibvirt
+        emptyDir:
+          medium: Memory
+{{- else }}
+      - name: instances
+        hostPath:
+          path: /var/lib/nova/instances
+      - name: ssh
+        hostPath:
+          path: /var/lib/nova/.ssh
+      - name: libvirt
+        hostPath:
+          path: /var/lib/libvirt
+      - name: run
+        hostPath:
+          path: /run
+      - name: host
+        hostPath:
+          path: /
+      - name: usr-share-qemu
+        hostPath:
+          path: /usr/share/qemu
+{{- end }}
+      - name: mnt
+{{- if not .Values.hypervisor_on_host }}
+        emptyDir:
+          medium: Memory
+{{- else }}
+        hostPath:
+          path: /var/lib/nova/mnt
+{{- end }}
+      - name: sys
+        hostPath:
+          path: /sys
+      - name: multipath-etc
+        hostPath:
+          path: /etc/multipath
+      - name: iscsi-etc
+        hostPath:
+          path: /etc/iscsi
+      - name: nfs
+        hostPath:
+          path: /var/lib/nfs
+      - name: nova-etc
+        projected:
+          sources:
+          - configMap:
+              name: {{ .Release.Name }}-etc
+              items:
+              - key: nova.conf
+                path: nova.conf
+              - key: logging.ini
+                path: logging.ini
+              - key: rootwrap.conf
+                path: rootwrap.conf
+              - key: compute.filters
+                path: rootwrap.conf.d/compute.filters
+          - configMap:
+              name: nova-compute-kvm
+              items:
+              - key: nova-compute.conf
+                path: nova-compute.conf
+          - secret:
+              name: {{ .Release.Name }}-etc
+              items:
+              - key: cell.conf
+                path: nova.conf.d/cell.conf
+              - key: keystoneauth-secrets.conf
+                path: nova.conf.d/keystoneauth-secrets.conf
+      - name: statsd-etc
+        projected:
+          sources:
+          - configMap:
+              name: {{ .Release.Name }}-etc
+              items:
+              - key:  statsd-exporter.yaml
+                path: statsd-exporter.yaml
```

Comment:
I do see settings for it, but I do not see the statsd container. Is that injected automatically later?